src/components/playback/playbackmanager.js

```diff
@@ -1872,6 +1872,7 @@ class PlaybackManager {
                 promise = apiClient.getEpisodes(firstItem.SeriesId || firstItem.Id, {
                     IsVirtualUnaired: false,
                     IsMissing: false,
+                    SortBy: options.shuffle ? 'Random' : 'SortName',
```

Comment:
If we get episodes in the correct order by default, it may be better to do this:

```suggestion
                    SortBy: options.shuffle ? 'Random' : undefined,
```

Reply:
Changed.

This is reasonable and seems to work. The server should respond with an appropriate default sort order when it's unspecified.